### ISSUE TITLE
relaysrv: out-of-date releases?

### ISSUE THREAD

**Issue by Kriechi** [syncthing/syncthing#8094]
The main repo has v1.18.5 as the latest release.

The last release of on the [relaysrv repository](https://github.com/syncthing/relaysrv/releases) is v1.15.0, but I suppose this is expected because the source code is in the main repo. A bit confusing or misleading nevertheless.

The last release of relaysrv as [Docker image on Docker Hub](https://hub.docker.com/r/syncthing/relaysrv/tags) is v1.18.0 about 6 months ago.

The last release of syncthing-relaysrv in the apt repository contains v1.18.3:
```
$ apt update
...
Hit:3 http://apt.syncthing.net syncthing InRelease
...
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
All packages are up to date.

$ apt info syncthing-relaysrv
Package: syncthing-relaysrv
Version: 1.18.3
...
APT-Sources: http://apt.syncthing.net syncthing/stable amd64 Packages
```

So here my question or bug report:
* Is syncthing and syncthing-relaysrv always released in tandem with the same version number?
* If yes, why is the Docker Hub image and the APT repository not updated?
* If no, should the main GitHub release, e.g., [this](https://github.com/syncthing/syncthing/releases/tag/v1.18.3), clearly indicate if relaysrv is bumped or not?

**Comment by bt90** [comment]
We should archive the relaysrv repo to avoid confusion.

**Comment by calmh** [comment]
The repo holds the releases, but the relaysrv changes very seldom and we don't do releases of it unless required. So I think it's in fact up to date.

**Comment by Kriechi** [comment]
@calmh ok, so relaysrv is only released when there really is a change. It would be great to make this explicit in the release notes.

This leaves the APT vs. DockerHub difference - is the APT release just "ahead without meaning" or is the DockerHub tag out of date?

**Comment by calmh** [comment]
There were some APT specific changes related to startup scripts, `on-upgrade` hooks and such.

**Comment by GermanCoding** [comment]
While we're on topic, I wanted to note that we're not actually fully up-to-date in apt:

AFAIK, latest change for strelaysrv & discosrv were indeed APT-specific changes (https://github.com/syncthing/syncthing/commit/e557ba82e7b624f8d0547dde0f2ec87d13a4099b), which landed in v1.18.4. However 1.18.4 was never released in apt. But the change wasn't really critical + any newer release should incorporate the fix, so I didn't bother reporting that so far.

**Comment by calmh** [comment]
New I've released all the things in all the places for 1.18.6, so we should be good
